Makefile

```diff
@@ -1,42 +1,30 @@
-venv:
```

Comment:
Since we no longer have a `venv` makefile target, we should also update [this section](https://github.com/ray-project/deltacat/blob/af8ce43f0f543afe5ef55d85777ce6df5a49713f/README-development.md?plain=1#L23-L34) of `README-development.md` and[ this line](https://github.com/ray-project/deltacat/blob/af8ce43f0f543afe5ef55d85777ce6df5a49713f/deltacat/examples/compactor/README.md?plain=1#L14) of the compactor example `README.md`.